src/types/global.ts

```diff
@@ -0,0 +1,9 @@
+declare module NodeJS {
+  interface Global {
+    IS_TEST: boolean
```

Comment:
why is this needed ?

Reply:
It is a flag you can check for in code while JEST is running it. It's not used now, but could be useful I think. Also as entrypoint for other global vars. Should I revert it?